tests/utils.spec.coffee

```diff
@@ -25,9 +25,7 @@ describe 'Utils:', ->
 			it 'should eventually become the configuration information', ->
 				promise = utils.getConfigPartitionInformationByType('raspberry-pi')
 				m.chai.expect(promise).to.eventually.become
-					partition:
-						primary: 4
-						logical: 1
+					partition: 5
```

Comment:
Would it make sense to also test for the old format?
`convertFilePathDefinition` seems to be able to handle that.

Reply:
I assume you mean _new_ format? This test is already using the old manifest format (above - this bit is asserting on the result, which should always be in the new format).

Good idea though, given that - I've added a test :+1: